templates/distribution/manifests/monitoring/resources/haproxy-scrapeConfig.yaml.tpl

```
@@ -1,4 +1,7 @@
-{{- if and (eq .spec.distribution.common.provider.type "none") .spec.kubernetes.loadBalancers.enabled }}
+
+{{- if and (eq .spec.distribution.common.provider.type "none") }}
```

this `and` is not needed anymore
```suggestion
{{- if eq .spec.distribution.common.provider.type "none" }}
```